app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html

```diff
@@ -79,7 +92,7 @@
     @cyaSection() {
         @cyaRow(
             messages(s"$key.sentToHmrc"),
-            Html(TimeUtils.govDisplayFormat(returnSummary.submissionDate)),
+            if(returnSummary.isRecentlyAmended){amendDate}else{Html(TimeUtils.govDisplayFormat(returnSummary.submissionDate))},
```

Comment:
you should be able to do something like: 
```scala
Html(TimeUtils.govDisplayFormat(returnSummary.lastyUpdatedDate.getOrElse(returnSummary.submissionDate)))
```
to avoid having to pattern match with `amendDate` up there with the `None` case